src/lib/unsafe-html.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import { directive, NodePart } from '../lit-html.js';
+
+const tmp = document.createElement('template');
+
+/**
+ * Renders the result as HTML, rather than text.
+ * 
+ * Note, this is unsafe to use with any user-provided input that hasn't been
+ * sanitized or escaped, as it may lead to cross-site-scripting
+ * vulnerabilities.
+ */
+export const unsafeHTML = (value: any) =>
+  directive((_part: NodePart) => {
+    tmp.innerHTML = value;
```

Comment:
Appears to make no difference. I'll go with the simpler approach of creating a new element each time.